packages/server/src/course/course.service.ts

```diff
@@ -234,4 +236,63 @@ export class CourseService {
       );
     }
   }
+
+  async getUserInfo(
+    courseId: number,
+    page: number,
+    pageSize: number,
```

Comment:
i think we can just have `pageSize` as a constant here right? idk if `getUserInfo` would be used anywhere else since this is specifically for the paginated endpoint

Reply:
that's what I had originally but I figured having page size associated with getting the user info might be good design for down the road but also idk. I guess this makes it so that you can change page size in the actual endpoint rather than in the service file?

Reply:
i think parametrizing the pageSize probably works in our favor for testing as well - like for those unit tests you can just set the page size to a really small number to check that pagination works